ocrd_models/ocrd_models/ocrd_file.py

```diff
@@ -117,6 +118,47 @@ def pageId(self, pageId):
         self.mets.set_physical_page_for_file(pageId, self)
 
 
+    @property
+    def loctype(self):
+        """
+        Get the ``LOCTYPE``.
+        """
+        el_FLocat = self._el.find('mets:FLocat', NS)
+        return '' if el_FLocat is None else el_FLocat.get('LOCTYPE')
+
+    @loctype.setter
+    def loctype(self, loctype):
+        """
+        Set the ``LOCTYPE``.
+        """
+        if loctype is None:
+            return
+        loctype = loctype.upper()
+        el_FLocat = self._el.find('mets:FLocat', NS)
+        if el_FLocat is None:
+            el_FLocat = ET.SubElement(self._el, TAG_METS_FLOCAT)
+        el_FLocat.set('LOCTYPE', loctype)
+        if loctype == 'OTHER':
+            self.otherloctype = 'FILE'
+        else:
+            self.otherloctype = None
+
+    @property
+    def otherloctype(self):
+        el_FLocat = self._el.find('mets:FLocat', NS)
+        return '' if el_FLocat is None else el_FLocat.get('OTHERLOCTYPE')
+
+    @otherloctype.setter
+    def otherloctype(self, otherloctype):
```

Comment:
I can set `OTHERLOCTYPE` without setting `LOCTYPE` to `OTHER`? Does this make sense? Especially considering that setting `LOCTYPE` to `OTHER` implicitly sets `OTHERLOCTYPE` to `FILE`...

Reply:
Good point. Fixed.